### PR TITLE
(feat) add member and event data from a JSON file 

### PR DESCRIPTION
**Pull request by aayushmau5**
Final Bug fixes attached as well. :)

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,3 +1,3 @@
 node_modules/
-src/node_modules/
 __sapper__/
+src/node_modules/@sapper/
```
Comment (humancalico) [inline .gitignore:3]: Is this change necessary?

**Review by humancalico — CHANGES_REQUESTED** [review]

File: .gitignore (hunk 1 above)
Comment (aayushmau5) [reply .gitignore:3]: Don't think so because it's not working.
